buildcache

```diff
@@ -302,7 +312,7 @@ if cache_dir:
     args = strace + sys.argv[1:]
 else:
     args = sys.argv[1:]
-strace = subprocess.Popen(args)
+strace = subprocess.Popen(args, bufsize=-1)
```

Comment:
Let's hope that solves the problem.

and someday, we'll have to write some native code that does what we need, rather than having strace spew lots of messages, from which we need a limited amount of information.